### PR TITLE
[WebAPI] Add 3 manual test cases for Google Play Purchase API.

### PR DESCRIPTION
**Pull request by yhe39**
Add 3 purchase api test file for Google Play and XiaoMi.
Update the suite.json to pack release package for the purchase test
Update the tests.full.xml and tests.xml.
Add the google/iapdemo.keystore for the package and sign the apk
Modify the README to indicate Google Play apk need sign and align
Correct the channel name in Navigator_iap_queryProductsInfo.html
Modify the configuration in Xiaomi/manifest.json
The Navigator_iap_purchase_undefined.html related bug: https://crosswalk-project.org/jira/browse/XWALK-6895

Impacted tests(approved): new 2, update 1, delete 0
Unit test platform: Crosswalk Project for Android 18.48.477.13
Unit test result summary: Pass 2, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-1552

### CODE REVIEW

**Comment by yhe39** [comment]
@Honry PTAL. thanks a lot!

File: webapi/webapi-iap-xwalk-tests/suite.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
       },
       "subapp-list": {
         "google": {
-          "apk-common-opts": "--enable-remote-debugging",
+          "apk-common-opts": "--release",
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/suite.json]: Is there a precondition to require user to sign the release app?

File: webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html [hunk 2]
```diff
@@ -29,62 +29,60 @@
...
-        }, "Check the purchaseState property");
-
-        at.done();
-      }).catch(function(e) {
-        assert_unreached("Should not be here, otherwise purchase failed!");
-        at.done();
+    options.channel = "Google";
+    options.debug = true;
+    options.key = "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2yD8XJgCd6IPtimK6P7OenXGixfMEoK8fo5H7ravu3vVhVuEcM3f/z/3EHHGTPlCHllFsv5FcCy84iPR2AqgEDpQLRUjaf62AX799h1rthVWxeHQN2l1DzQpCVeWNiPPQLDE6zJ5Cjhr8e0hgFlxNe8iLJNssEFXZQX+xUzskNbn0CidBquIG3DLolQjJvrxxB6OizZ7BrXQ5LvVLIIQ96W2jvDd6H20NtGi88RkiXT0ayKVR7Yi1SNamu2K1OnlnEQBL/cT63hkBW0osAX1drZruMDcBHFC9SKHYynV/9dUEkUaBQ37ag0i2AEDeipdXfU7F8mUvYLEA6w1LlUuJwIDAQAB";
+    queryProducts = ["org.xwalk.apple", "org.xwalk.orange"];
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase.html]: Seems this variable is never called.

File: webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase_undefined.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  </div>
+  <div id="result" style="width:100%; height:100px; word-wrap: break-word; word-break: normal;"></div>
+</div>
+<script type='text/javascript'>
+  function purchasetestWrong() {
+    var html = "</div>";
+    var notice = "";
+    var options = {};
+
+    options.channel = "Google";
```
Comment (Honry) [inline webapi/webapi-iap-xwalk-tests/iap/Navigator_iap_purchase_undefined.html]: What if it is Xiaomi channel?

**Comment by yhe39** [comment]
@Honry change files according to your comments. PTAL. thanks a lot!

**Comment by yhe39** [comment]
Update the expected result format. Add the related bug information to the comment.

**Comment by Honry** [comment]
LGTM

**Pull request merged**